spec/system/reporting/snapshot_reports_spec.rb

```diff
@@ -22,19 +22,6 @@
       expect(page.title).to have_text 'Workload report'
     end
 
-    it 'shows the correct column headers' do
-      expected_headers = ['', 'Received', 'Closed', '0 days', '1 day', '2 days', '3 days', '4-9 days', 'Total']
-
-      page.all('table thead tr.colgroup-details th').each_with_index do |el, i|
-        expect(el).to have_content expected_headers[i]
-      end
-    end
-
-    it 'shows the correct row headers for each work stream' do
-      column = all('table tbody tr th:first-child').map(&:text)
-      expect(column).to eq ['CAT 1', 'CAT 2', 'Extradition', 'Total']
-    end
-
```

Comment:
Why was this deleted?

Reply:
It is tested in the workload_report spec and does not need to be tested here. This spec is for the general snapshot functionality (paging by time etc) and should not be testing the contents of the workload report.